README.md

```diff
@@ -46,3 +46,15 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+The worst case: 
+the initialization function can be ran |v| number of times where |v| is the number of nodes in the graph. (it actually runs |v| + |v| times but only the 2nd |v| has more functioning to do). This |v| is only here because it allows us to start at every city.
+
+The actuall function itself worst case runs over every entry in the distance_matrix so |v|. for each of those iterations it tries every possible smaller iteration |v|-1 untill it gets to the base case of 2 where it just returns the distances. so |v| * |v|-1 * |v|-2 *...* 2 $\in n!$
```

Comment:
Ok, but you're memoizing on sorted subsets, right? That is, `1,2,3` with start `0` and `3,2,1` with start `0` amount to the same thing. So are you really checking all permutations?